llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -3129,279 +3129,279 @@ foreach ta = [v2f16, v2bf16, v2i16, v4i8, i32] in {
 // and then cvt to floating-point.
 
 // sint -> f16
-def : Pat<(f16 (sint_to_fp Int1Regs:$a)),
+def : Pat<(f16 (sint_to_fp i1:$a)),
           (CVT_f16_s32 (SELP_s32ii -1, 0, Int1Regs:$a), CvtRN)>;
```

Comment:
I noticed that some top-level patterns use regclasses on the RHS, while others were converted to use types.
Is this intentional or overlooked?